_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,236 @@
+  ensure the counterparty will be satisfied even if they're using a
+  different fee estimation algorithm.
+
+    BOLT2 also allows channels to route up to 483 payments
+    simultaneously, each of which requires a 43 vbyte P2WSH output, for
+    a total of about 20,000 vbytes of data that needs to be added to the
+    chain relatively quickly---meaning it may need to pay a high feerate.
+    If that feerate is five times higher than strictly necessary, this
+    can easily result in paying more than a hundred dollars in
+    transaction fees.  Additionally, if the commitment transaction is
+    confirmed, the HTLCs then need to be settled (again using
+    time-sensitive transactions that may need to pay a high feerate).
+    If the victim was the party routing those payments outbound, they'll
+    need to pay an additional transaction fee to recover each payment,
+    which could make the attack two or three times more costly for them.
+
+    Fees are paid to miners, so there's no direct motivation to perform
+    this attack, but if the attacker has the means to quickly contact
+    the victim, it's possible they can offer to settle the HTLCs
+    offchain in return for a ransom, allowing the attacker to profit.
+
+    Pickhardt concludes his email with several ideas for addressing the
+    problem, none of which he finds completely satisfactory.  The
+    mitigation originally implemented in Eclair and later implemented in
+    C-Lightning (see [Newsletter #59][news59 cl30]) is for LN nodes to limit the
+    number of pending payments, keeping transactions small and total
+    fees low.  Another mitigation in development are [anchor
+    outputs][topic anchor outputs] which allow feerates to be selected
+    when the channel is closed---eliminating the need to overestimate
+    fees in order to prevent premature channel closures.  Several other
+    ideas are mentioned, but Pickhardt asks readers to contemplate the
+    problem and suggest any other possible solutions.
+
+- **Continued discussion about LN atomicity attack:** Bastien Teinturier
+  [posted][teinturier post] to the Lightning-Dev mailing list with a
+  link to a [detailed description][teinturier gist] of the LN commitment
+  protocol, it's weaknesses, and proposals to address those weaknesses.
+  The document teaches readers everything they should need to know to
+  understand the attack against LN atomicity described in [Newsletter
+  #95][news95 ln attack] as well as several proposed mitigations.  The
+  document's clear writing tied together several fragmented threads from
+  previous discussions.  This lead to renewed evaluation of several
+  previously-proposed solutions, including concerns about the
+  effectiveness of the "alternative anchor proposal" and a [suggested
+  procedure][zmn procedure] to use the [pay-for-signature][] scriptless
+  script to trustlessly pay a third-party for the final signature needed
```

Comment:
remove hyphens? "previously proposed" and "third party" (since the latter isn't being used as a combined adjective)